src/lib/jsonParseV2/parseEntity.cpp

```diff
@@ -49,6 +49,9 @@ using namespace rapidjson;
 *
 * In the case of /v2/entities/<eid>, the entityId of 'Entity* eP' is set in
 * the service routine postEntity.
+*
+* Also, if the URI param 'options' includes the value 'keyValues', then the
+* parse changes to accept 'unknown fields' as attributes inside entities.
```

Comment:
True.
I guess I was thinking about the way I wanted to implement the metadatas when I wrote this.